_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -0,0 +1,192 @@
+## Deploying on AWS EC2 Ubuntu using PostgreSQL
+
+Here are the steps:
+
+1. Log into your AWS account or create a new one [AWS Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+2. Go to AWS EC2 Dashboard
+3. Launch Instances
+4. On **Application and OS Images (Amazon Machine Image)** select `Ubuntu`
+5. On **Instance Type** select `t2.micro` (is ok for testing and small projects which is Free tier eligible)
+6. Create or select an existing `key pair`. (If you create a new one click `Download Key Pair`)
+7. Click `Launch Instance`
+8. On **EC2 Dashboard** select the new created instance and click `Security`
+9. On **Security Groups**, click on the security group _i.e.:sg-0fg55kj5588d63_
+10. On **Inbound Rules** tab, click `Edit Inbound Rules`
+11. Click `Add rule` and select `PostgreSQL` from the dropdown menu and `Anywhere-IPv4`.
+12. Click `Save rules`
+13. On **EC2 Dashboard** select the new created instance and click `Connect`
+14. Click `SSH Client` and follow the instructions
+
+Once logged into the ec2 instance we perform the following tasks:
+
+### Install Libraries and Dependencies
+Update the local package manager `apt`
+```bash
+sudo apt update
+```
+Install [NodeJS](https://nodejs.org)
+```bash
+sudo apt install nodejs
+```
+Check the install was ok, you should see the version installed. _i.e: v16.15.0_
```

Comment:
I would not add a specific version here, this is something we'd have to maintain in the docs, because the version will look pretty old pretty soon. Maybe remove "`_i.e: v16.15.0_`"